nautobot/extras/tables.py

```diff
@@ -78,11 +78,8 @@
 """
 
 TAGGED_ITEM = """
-{% if value.get_absolute_url %}
-    <a href="{{ value.get_absolute_url }}">{{ value }}</a>
-{% else %}
-    {{ value }}
-{% endif %}
+{% load helpers %}
+{{ value|hyperlinked_object }}
 """
```

Comment:
And this (in combination with the other commented diff above) fixes the error when rendering the Tags view if some of the tagged objects don't have a UI URL.